jkube-kit/jkube-kit-wildfly-jar/src/main/java/org/eclipse/jkube/wildfly/jar/enricher/WildflyJARHealthCheckEnricher.java

```diff
@@ -67,12 +79,20 @@ private enum Config implements Configs.Config {
 
     @Override
     protected Probe getReadinessProbe() {
-        return discoverWildflyJARHealthCheck(true);
+        return discoverWildflyJARHealthCheck(getConfig(Config.READINESS_PATH), asInt(getConfig(Config.READINESS_INITIAL_DELAY)), asInt(getConfig(Config.FAILURE_THRESHOLD)), asInt(getConfig(Config.SUCCESS_THRESHOLD)), asInt(getConfig(Config.READINESS_PERIOD_SECONDS)));
```

Comment:
Considering that in the end we're passing the values extracted from configurations by following the same pattern in all of the `discoverWildflyJARHealthCheck` method calls, why not just pass the Configuration reference?

```suggestion
        return discoverWildflyJARHealthCheck(Config.READINESS_PATH, Config.READINESS_INITIAL_DELAY, Config.FAILURE_THRESHOLD, Config.SUCCESS_THRESHOLD, Config.READINESS_PERIOD_SECONDS);
```

Reply:
Are you going to address this? Is there something else I can add or elaborate on to clarify what I mean?

Reply:
> Are you going to address this?

Yes


> Is there something else I can add or elaborate on to clarify what I mean?

No